recipe/bld.bat

```diff
@@ -1,23 +1,17 @@
-(
-echo [DEFAULT]
-echo library_dirs = %LIBRARY_LIB%
-echo include_dirs = %LIBRARY_INC%
-echo [lapack]
-echo libraries = lapack,blas
-echo [blas]
-echo libraries = cblas,blas
-echo [cblas]
-echo libraries = cblas,blas
-) > site.cfg
+@echo on
 
-set "NPY_LAPACK_ORDER=lapack"
-set "NPY_BLAS_ORDER=blas"
+mkdir builddir
 
-%PYTHON% -m pip install --no-deps --ignore-installed -v .
-if errorlevel 1 exit 1
+:: -wnx flags mean: --wheel --no-isolation --skip-dependency-check
+%PYTHON% -m build -w -n -x ^
+    -Cbuilddir=builddir ^
+    -Csetup-args=-Dblas=blas ^
+    -Csetup-args=-Dlapack=lapack
+if %ERRORLEVEL% neq 0 exit 1
 
-XCOPY %RECIPE_DIR%\f2py.bat %SCRIPTS% /s /e
-if errorlevel 1 exit 1
-
-del %SCRIPTS%\f2py.exe
-if errorlevel 1 exit 1
```

Comment:
@conda-forge/numpy 

Not sure what the existing setup for f2py on windows was here, resp. what constraints we have to pay attention to. Currently, `meta.yaml` sets a windows-specific entry point, and the artefact (for 1.25.2) contains
```
Scripts/f2py-script.py
Scripts/f2py.bat
Scripts/f2py.exe
```
in `%PREFIX%`.

Calling `f2py -h` still works successfully in this PR, but I'm not sure if we need to pay attention to something else here.

Reply:
Perhaps @ocefpaf knows (or remembers from https://github.com/conda-forge/numpy-feedstock/pull/96).

Reply:
I think I copied that from the anaconda recipe. We probably don't need it anymore. I think it was a limitation with how -build handled the scripts.